OpenSim/Simulation/Manager/Manager.h

```diff
@@ -167,25 +168,64 @@ class OSIMSIMULATION_API Manager
     { _writeToStorage =  writeToStorage; }
 
     // Integrator
-    enum class Integrator {
+    enum class IntegratorMethod {
+        /** For details, see SimTK::CPodesIntegrator. */
         CPodes,
+        
+        /** For details, see SimTK::ExplicitEulerIntegrator. */
         ExplicitEuler,
+        
+        /** For details, see SimTK::RungeKutta2Integrator. */
         RungeKutta2,
+        
+        /** For details, see SimTK::RungeKutta3Integrator. */
         RungeKutta3,
+        
+        /** For details, see SimTK::RungeKuttaFeldbergIntegrator. */
         RungeKuttaFeldberg,
+
+        /** For details, see SimTK::RungeKuttaMersonIntegrator. */
         RungeKuttaMerson,
+
         //SemiExplicitEuler, no error ctrl, requires fixed stepSize arg on construction
+
+        /** For details, see SimTK::SemiExplcitEuler2Integrator. */
         SemiExplcitEuler2,
+
+        /** For details, see SimTK::VerletIntegrator. */
         Verlet
     };
-    void setIntegrator(Integrator integMethod);
+
+    /** Sets the type of integrator used via IntegratorMethod enum. The 
+      * integrator will be set to its default options. For MATLAB/Python
+      * users, see `setIntegrator(const std::string&)` instead.*/
+    void setIntegrator(IntegratorMethod integMethod);
```

Comment:
What do you think about naming this method `setIntegratorMethod()`? Right now, there is a `getIntegrator()` that returns the actual integrator.

Reply:
I think that'd be a fine change. If you're leaning that way I'd be happy to do so.